.agents/skills/qodo-pr-resolver/resources/providers.md

```
@@ -0,0 +1,463 @@
+- **Verify:**
+  ```bash
+  glab --version && glab auth status
+  ```
+
+### Bitbucket
+
+**Authentication:** Bitbucket REST API with an App Password (there is no official `bb` CLI)
+- Create an App Password: Bitbucket → **Settings → App passwords**
+  - Required scopes: **Repositories: Read**, **Pull requests: Read, Write**
+- **Qodo config** (`~/.qodo/config.json`) — store credentials persistently:
+  ```json
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)